src/DynamoCore/Scheduler/AsyncTask.cs

```diff
@@ -195,6 +196,9 @@ internal bool Execute()
             finally
             {
                 ExecutionEndTime = scheduler.NextTimeStamp;
+
+                var elapsedTime = new TimeSpan(ExecutionEndTime.TickCount - ExecutionStartTime.TickCount);
+                Analytics.TrackTimedEvent(Categories.Performance, Actions.UpdateGraphAsyncTask.ToString(), elapsedTime, "Dynamo graph execution time");
             }
```

Comment:
@QilongTang is there a way to retrieve the host info here? One of the asks for this task is also to determine whether these graph execution requests are being made from Dynamo (host or sandbox) or Player, etc.?

Reply:
HostInfo is available from DynamoModel, but I think when we discuss that with @deepakanand earlier, he prefer to not use `HostAnalyticsInfo` object to identify player. Maybe he has some other thoughts in mind about implementation